docs/howtos/another_kolibri_instance.md

```diff
@@ -0,0 +1,55 @@
+## Running Another Kolibri Instance Alongside the Development Server
+
+This guide will walk you through the process of setting up and running another instance of Kolibri alongside your development server using the `pex` executable.
+
+## Introduction
+
+As Kolibri's features continue to expand into remote content browsing, it's often necessary to test and experiment with another Kolibri instance running alongside your development server. One effective approach is to use the `pex` executable. This workflow is straightforward and can be employed independently of ZeroTier or even internet network access. Documenting these steps will provide a handy reference for both new team members and contributors.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- A recent `pex` executable of Kolibri. You can download this from the Build Assets page of a workflow build.
+
+## Steps
+
+- **Locate the .pex executable:**
+
+  - Navigate to the [Kolibri GitHub repository](https://github.com/learningequality/kolibri).
+  - Click on the "Actions" tab at the top of the repository.
+  - Select a workflow build from the list.
+  - Scroll down the workflow build page to find the "Artifacts" section. In this section, you will find the `.pex` file that you need to download. Click on the `.pex` file to initiate the download.
+
+- **Save and unzip the .pex file:**
+
+  Save the downloaded `.pex` file to a suitable location on your machine. Unzip the downloaded `pex` file to a folder where you want to run the additional Kolibri instance from.
+
+- **Run another Kolibri instance:**
+
+  Open your terminal and navigate to the folder where you unzipped the `pex` file. Use the following command to start another Kolibri instance:
+
+  ```sh
+  KOLIBRI_HOME="kolibri-home" python <filename>.pex start
+  ```
+  Replace `<filename>` with the actual filename of the `pex` executable. You can choose any location for `KOLIBRI_HOME`, but ensure it's set so that it doesn't override your default `.kolibri` directory.
+
+- **Complete initial setup:**
+
+  In the terminal, you'll find the URL of the new Kolibri instance. Open the URL in your browser and complete the initial device setup as you would for a regular Kolibri instance. Additionally, import a few resources from desired channels.
+
+- **Run your development server:**
+
+  Once the additional Kolibri instance is up and running, start your development server as usual. You should now see the new device on your network.
+
+- **Stop the other Kolibri instance:**
+
+  When you're done testing, you can stop the additional Kolibri instance using the following command:
+  ```sh
+  KOLIBRI_HOME="kolibri-home" python <filename>.pex stop

```

Comment:
Ah one more thing, I think we actually won't need `KOLIBRI_HOME="kolibri-home"` when running the `stop` command, so it can say `python <filename>.pex stop`